doc/modules/gaussian_process.rst

```diff
@@ -487,7 +487,7 @@ number of dimensions as the inputs :math:`x` (anisotropic variant of the kernel)
 The kernel is given by:
 
 .. math::
-   k(x_i, x_j) = \text{exp}\left(-\frac{1}{2} d(x_i / l, x_j / l)^2\right)
+   k(x_i, x_j) = \text{exp}\left(- \frac{d(x_i, x_j)^2}{2l^2} \right)
```

Comment:
could you please leave some comments around these changes and explain them?

Reply:
I used this formulation for the RBF so that it agrees with the one used by the [Kernel Cookbook](https://www.cs.toronto.edu/~duvenaud/cookbook/) and the [Gaussian Process book by Rasmussen, p. 84/85](http://www.gaussianprocess.org/gpml/chapters/RW.pdf) since especially the later is the most-known reference for Gaussian processes.

For euclidean distance, these formulas are the same. I'm actually not sure about other distances, I also didn't see anyone else using other distances. Could change the $d(x_i, x_j)^2$ to $(x_i - x_j)^2$

Reply:
Where should I leave these comments? Just here or in the rst-File? (Can you leave comments in an rst-File?)

Reply:
Yes you can leave comments there. But I meant here. I'm happy with what you wrote :)

Reply:
I would note that `d(., .)` is the distance between them though

Reply:
While adding the note that `d(.,.)` is the Euclidean distance, I noticed that the[Matern kernel](https://scikit-learn.org/stable/modules/gaussian_process.html#matern-kernel) uses a bunch of other notation that's not explained. Cross-referencing with other sources I could find out that $K_\nu$ is a modified Bessel function and $\Gamma$ is the gamma function. However, I have no idea what the small $\gamma$ stands for. With short googling, I couldn't find a reference that uses $\gamma. It is also not a function parameter.
Similarly for $\sigma^2$: It is not commonly used in other references (though I found some that use it) and it's not a function parameter, that is in the code this is always 1.
I would thus propose to omit both $\sigma$ and $\gamma$ from the formulas.